zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1382,6 +1382,59 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /decision-instances/{decisionInstanceKey}:
+    get:
+      tags:
+        - Decision instance
+      summary: Get decision instance by key (alpha)
```

Comment:
Do we need a period at the end here?

Reply:
No, all summaries in the file do not end with a period